inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -291,11 +290,8 @@ func createUnencryptedTemplate(templateDir string) string {
 }
 
 func getServerIp() string {
-	serverIp := promptString("\nPlease enter the server IP:")
-	if net.ParseIP(serverIp) == nil {
-		log.Fatalf("Invalid IP address provided.")
-	}
-	return serverIp
+	serverIPName := promptString("\nPlease enter the server IP or hostname")
```

Comment:
I think if the first character starts with a number, we should still do the ParseIP check.  I'm not sure if they have a hostname checker in that same library.  It's worth checking.

Reply:
@tsirlapu this might help with checking for valid host or ip: